cmd/server/assets/apikeys/index.html

```diff
@@ -2,6 +2,12 @@
 
 {{$authApp := .authApp}}
 
+{{$currentMembership := .currentMembership}}
+{{$canWrite := false}}
```

Comment:
`{{$canWrite := .currentMembership.Can rbac.APIKeyWrite}}`

simplify?

Reply:
For some reason, that wasn't working, but I tried again and it's working. /shrug.